app/src/main/java/io/gnosis/safe/HeimdallApplication.kt

```diff
@@ -42,3 +49,26 @@ class HeimdallApplication : MultiDexApplication(), ComponentProvider {
         }
     }
 }
+
+private class CrashReportingTree : Timber.Tree() {
```

Comment:
Is this really a crash reporting tree? Isn't it more an exception reporting tree? Crashes are already reported. This is about reporting otherwise swallowed exceptions?

Reply:
yes, Should I rename it to ErrorReportingTree?